extensions/spaces/CHANGELOG.md

```diff
@@ -2,5 +2,13 @@
 
 ## [Initial Version] - 2022-06-21
 
-If you have the Spaces app, you can access it straight from Raycast. This release contains one command:
-Create: The command opens a windows to create a new space in the app.
+If you have the Spaces app, you can access it straight from Raycast.
+This release contains one command:
+
+- Create Space: The command opens a windows to create a new space in the app.
+
+## 1.1 - 2022-6-22
```

Comment:
To keep coherent and follow our changelog convention, you could replace 1.1 with [New Additions]
```suggestion
## [New Additions] - 2022-6-22
```